src/StatsdClient/Worker/Waiter.cs

```diff
@@ -1,13 +1,16 @@
 using System;
-using System.Threading.Tasks;
 
 namespace StatsdClient.Worker
 {
     internal class Waiter : IWaiter
     {
         public void Wait(TimeSpan duration)
         {
-            Task.Delay(duration).Wait();
+#if NETSTANDARD1_3
+            System.Threading.Tasks.Task.Delay(duration).Wait();
```

Comment:
Should this use `ConfigureAwait(false)`?
```suggestion
            System.Threading.Tasks.Task.Delay(duration).ConfigureAwait(false).GetAwaiter().GetResult();
```